index.bs

```diff
@@ -297,6 +301,14 @@ switches from "hidden" to "visible", the user agent MAY run these steps:
 2. If {{pictureInPictureElement}} is <a>autoPictureInPictureElement</a>, run
     the <a>exit Picture-in-Picture algorithm</a>.
 
+## Interaction with Fullscreen ## {#fullscreen}
+
+It is RECOMMENDED to run the <a>exit Picture-in-Picture algorithm</a> when the
+{{pictureInPictureElement}} <a>fullscreen flag</a> is set.
+
+It is RECOMMENDED to <a>exit fullscreen</a> when the <a>request Picture-in-Picture
+algorithm</a> is invoked with <a>fullscreenElement</a>.
```

Comment:
Should that bit simply be added in the steps instead of being out of bounds? This section could mention that it's mirroring the request pip algorithm?

Reply:
Done in https://github.com/w3c/picture-in-picture/pull/185/commits/b05fef83ed67079c08792bf43801cbab944e293a